docs/style-banner.html

```diff
@@ -583,15 +583,14 @@ <h3 class="item-subtitle">
 
       
       
-      <pre class="hljs-pre"><code class="lang-css">[data-region=&#39;banner&#39;] {
+      <pre class="hljs-pre"><code class="lang-css">[data-region=banner] {
```

Comment:
@mirisuzanne Whoa -- these changes are weird.

![Screenshot 2020-04-24 18 30 14](https://user-images.githubusercontent.com/552316/80261962-f6db4d00-8659-11ea-8af3-9936f885fe8f.png)

Reply:
Yeah, that's valid - and has the same meaning in CSS. Dart Sass is more aggressive about optimizing output like that.

Reply:
🤯